<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Single-cell Regulatory Occupancy Archive in Dementia | Zechuan Shi </title> <meta name="author" content="Zechuan Shi"> <meta name="description" content="scROAD - A database offers comprehensive information on single-cell cCRE transcription factor occupancy data"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?29a31447c2ea9b76de9d010225a62982"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://rootze.github.io/projects/scROAD_project/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Zechuan</span> Shi </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Single-cell Regulatory Occupancy Archive in Dementia</h1> <p class="post-description">scROAD - A database offers comprehensive information on single-cell cCRE transcription factor occupancy data</p> </header> <article> <p>Database: Single-cell Regulatory Occupancy Archive in Dementia – <a href="https://swaruplab.bio.uci.edu/scROAD/" rel="external nofollow noopener" target="_blank">scROAD</a></p> <h3 id="how-can-scatac-seq-data-be-utilized-to-study-gene-regulation-and-chromatin-accessibility-in-diseases">How can scATAC-seq data be utilized to study gene regulation and chromatin accessibility in diseases?</h3> <p>scATAC-seq data enables us to explore chromatin accessibility at a single-cell level, allowing for the identification of regulatory elements that are active in specific cell types. This technique is particularly powerful for investigating transcription factor binding sites (TFBS) and understanding gene regulatory networks in the context of complex diseases like Alzheimer’s Disease (AD) and Pick’s Disease (PiD).</p> <p>Our comprehensive approach integrates genomic data from various sources, including GWAS fine-mapping, single-nucleus ATAC-seq (snATAC-seq), and single-nucleus RNA-seq (snRNA-seq), to uncover the genetic underpinnings of neurodegenerative diseases. By mapping disease-associated loci to specific cell types, we can investigate the functional alternations due to genetic variants in disease contexts, such as AD and PiD.</p> <p>I developed a wrapper package, <a href="https://github.com/rootze/scROAD" rel="external nofollow noopener" target="_blank">scROAD</a>, designed to streamline the extracting information from single-cell data generated by CellRanger ATAC, Seurat, and Signac into Cicero and TOBIAS for comprehensive analysis. We performed single cell co-accessibility analyses using <a href="https://cole-trapnell-lab.github.io/cicero-release/docs_m3/" rel="external nofollow noopener" target="_blank">Cicero</a> to construct putative cis-regulatory enhancer-promoter links. Additionally, with the help from <a href="https://github.com/loosolab/TOBIAS" rel="external nofollow noopener" target="_blank">TOBIAS</a> package, we can further explore transcription factor (TF) binding occupancy in ATAC-seq. This analysis allows us to detect differences in TF binding between disease and control samples, providing insights into how regulatory mechanisms are altered in specific cell types. By integrating TF binding data with co-accessibility analyses to create this scROAD interactive database, users can easily explore transcription factor binding activity and their implications in disease, providing a valuable resource for understanding gene regulation in neurodegeneration.</p> <h3 id="about-the-database">About the Database</h3> <p>This database <a class="citation" href="#shi_single-nucleus_2024">(Shi et al., 2024)</a> offers comprehensive information on single-cell cCRE transcription factor occupancy data generated from snATAC-seq analysis of human postmortem prefrontal cortex (PFC) tissue. The data specifically focuses on Alzheimer’s Disease and Pick’s Disease. For a more in-depth understanding of the database’s purpose and contents, please refer to the following publication. If you have any further questions, feel free to contact the principal investigator.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/projects_img/GWAS_snATAC_scROAD.png" sizes="95vw"></source> <img src="/assets/img/projects_img/GWAS_snATAC_scROAD.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="GWAS snATAC scCis-TF Occupancy" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Fig.1 GWAS snATAC scCis-TF occupancy plot showcasing key transcription factor binding sites </div> <h3 id="unveiling-the-power-of-tf-footprinting-a-new-frontier-in-regulatory-network-analysis-for-neurodegenerative-diseases">Unveiling the Power of TF Footprinting: A New Frontier in Regulatory Network Analysis for Neurodegenerative Diseases</h3> <h4 id="introduction">Introduction</h4> <p>Understanding transcription factor (TF) binding is essential for unraveling the complex regulatory networks underlying gene expression. Tools like scATAC-seq offer unprecedented opportunities to study chromatin accessibility at the single-cell level, shedding light on gene regulation and its dysregulation in disease. However, existing methods for analyzing TF networks often fall short of distinguishing functional TF binding events from non-functional motifs. In this blog, we explore a novel approach leveraging TF footprinting to overcome these limitations, offering new insights into neurodegenerative diseases like Alzheimer’s Disease (AD) and Pick’s Disease (PiD).</p> <h4 id="challenges-in-current-methods">Challenges in Current Methods</h4> <p>Widely used methods such as <a href="https://doi.org/10.1038/nmeth.4463" rel="external nofollow noopener" target="_blank">Aibar et al., 2017</a> SCENIC and <a href="https://doi.org/10.1038/s41592-023-01938-4" rel="external nofollow noopener" target="_blank">Bravo González-Blas et al., 2023</a> SCENIC+ rely on conventional motif enrichment analyses to construct gene regulatory networks (GRNs). While SCENIC uses promoter regions and co-expression patterns from scRNA-seq data, SCENIC+ extends this by integrating scATAC-seq and scRNA-seq data to link enhancers to target genes.</p> <p>However, these methods have notable limitations:</p> <ul> <li> <p>They infer TF activity indirectly through overrepresented motifs, which can lead to false positives.</p> </li> <li> <p>They struggle to distinguish functional enhancer-TF interactions from non-functional motifs.</p> </li> </ul> <p>For instance, a multi-omic study in <a href="https://doi.org/10.1038/s41586-024-07606-7" rel="external nofollow noopener" target="_blank">Mathys et al., 2024</a> AD demonstrated the utility of SCENIC in constructing cell-type-level TF regulators in AD snRNA-seq data. SCENIC+, on the other hand, builds on this by using pycistarget, a wrapper for HOMER, for enhancer motif enrichment. Unfortunately, these approaches often miss the mark in identifying true TF binding events.</p> <h4 id="limitations-of-open-chromatin-as-a-regulatory-marker">Limitations of Open Chromatin as a Regulatory Marker</h4> <p>A common assumption in chromatin accessibility studies is that open chromatin regions correlate with active regulatory elements. However, recent findings challenge this notion.</p> <p>Studies like <a href="https://doi.org/10.1016/j.cell.2023.08.040" rel="external nofollow noopener" target="_blank">Xiong et al., 2023</a> revealed that increased chromatin accessibility in neurodegenerative diseases often reflects chromatin relaxation rather than functional regulation. Similarly, <a href="https://doi.org/10.1038/nn.3639" rel="external nofollow noopener" target="_blank">Frost et al., 2014</a> observed chromatin relaxation and heterochromatin loss in tauopathies such as PiD and AD. These ‘relaxed’ chromatin regions may not indicate meaningful regulatory activity, as they often lack functional TF binding.</p> <p>Further complicating the picture, <a href="https://doi.org/10.1016/j.celrep.2017.05.003" rel="external nofollow noopener" target="_blank">Baek et al., 2017</a> reported that 80% of TF binding motifs do not show measurable footprints, suggesting that open chromatin alone is an unreliable indicator of active regulation. Methods like SCENIC and SCENIC+ may still infer TF activity in these regions, potentially resulting in false positives.</p> <p>This phenomenon is illustrated in Fig. 2 below, where we depict two scenarios: (1) open chromatin regions with TF binding (‘functional regulation’) and (2) open chromatin regions without TF binding (‘chromatin relaxation’).</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="https://raw.githubusercontent.com/rootze/scROAD/main/images/TF_Footprinting_Scenario.png" sizes="95vw"></source> <img src="https://raw.githubusercontent.com/rootze/scROAD/main/images/TF_Footprinting_Scenario.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="TF Footprinting Scenario" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Fig. 2 shows two key scenarios of transcription factor binding in open chromatin regions. Scenario 1 highlights regions where open chromatin coincides with functional TF binding, while Scenario 2 depicts regions with open chromatin but without active TF binding, indicating chromatin relaxation rather than regulation. </div> <h4 id="a-funtional-tf-footprinting-approach-on-disease-related-regulatory-network">A Funtional TF Footprinting Approach on Disease Related Regulatory Network</h4> <p>To address these challenges, we reimplement a bulk ATAC method on single cell data that incorporates TF footprinting and motif-flanking accessibility using the <a href="https://doi.org/10.1038/s41467-020-18035-1" rel="external nofollow noopener" target="_blank">TOBIAS</a> package. This approach offers a more accurate and confident way to identify active TF binding events.</p> <ul> <li> <p>TOBIAS calculates TF occupancy across all accessible chromatin regions, allowing us to:</p> <ul> <li> <p>Distinguish TF-occupied enhancers from non-functional motifs.</p> </li> <li> <p>Assess footprinting at binding sites and motif-flanking accessibility.</p> </li> <li> <p>Compare TF occupancy between disease and control conditions.</p> </li> </ul> </li> <li> <p>Key Advantages of the TF Footprinting Approach on Disease Related Regulatory Network:</p> <ul> <li> <p>Direct Measurement of TF Occupancy: Unlike SCENIC and SCENIC+, which rely on motif enrichment, our method directly measures TF binding, reducing false positives.</p> </li> <li> <p>Higher-Resolution Insights: By distinguishing functional enhancer-TF interactions from non-functional motifs, we provide a more accurate view of regulatory networks.</p> </li> <li> <p>Enhanced Integration: Combining snATAC-seq and snRNA-seq data allows for comprehensive analyses of TF-mediated regulation in specific cell types.</p> </li> </ul> </li> </ul> <h4 id="implications-for-neurodegenerative-disease-research">Implications for Neurodegenerative Disease Research</h4> <p>This method holds significant promise for advancing our understanding of transcriptional regulation in neurodegenerative diseases like AD and PiD. By uncovering true TF binding events and regulatory interactions, we can identify novel therapeutic targets and gain new insights into disease mechanisms.</p> <p>For example, this approach can help clarify the role of chromatin accessibility changes in neurodegeneration, distinguishing functional regulation from non-functional chromatin relaxation. Additionally, it provides a valuable resource for exploring TF activity and its implications in disease contexts.</p> <h4 id="conclusion">Conclusion</h4> <p>This funtional TF footprinting approach represents a major step forward in regulatory network analysis, addressing the limitations of conventional methods like SCENIC and SCENIC+. By leveraging tools like TOBIAS to measure TF occupancy directly, we can distinguish functional regulatory events from non-functional motifs, providing deeper insights into gene regulation in neurodegenerative diseases. As we continue to refine these methods, they will undoubtedly play a critical role in shaping the future of genomic research and therapeutic discovery.</p> <h4 id="papers-mentioned-above">Papers Mentioned Above</h4> <p>S. Aibar, C. B. González-Blas, T. Moerman, V. A. Huynh-Thu, H. Imrichova, G. Hulselmans, F. Rambow, J. C. Marine, P. Geurts, J. Aerts, et al., SCENIC: single-cell regulatory network inference and clustering. Nat. Methods 14, 1083–1086 (2017). DOI: https://doi.org/10.1038/nmeth.4463</p> <p>C. Bravo González-Blas, S. De Winter, G. Hulselmans, N. Hecker, I. Matetovici, V. Christiaens, S. Poovathingal, J. Wouters, S. Aibar, S. Aerts, SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks. Nat. Methods 20, 1355–1367 (2023). DOI: https://doi.org/10.1038/s41592-023-01938-4</p> <p>H. Mathys, C. A. Boix, L. A. Akay, Z. Xia, J. Davila-Velderrain, A. P. Ng, X. Jiang, G. Abdelhady, K. Galani, J. Mantero, N. Band, B. T. James, S. Babu, F. Galiana-Melendez, K. Louderback, D. Prokopenko, R. E. Tanzi, D. A. Bennett, L. H. Tsai, M. Kellis, Single-cell multiregion dissection of Alzheimer’s disease. Nature 632, 858–868 (2024). DOI: https://doi.org/10.1038/s41586-024-07606-7</p> <p>X. Xiong, B. T. James, C. A. Boix, Y. P. Park, K. Galani, M. B. Victor, N. Sun, L. Hou, L. L. Ho, J. Mantero, A. N. Scannail, V. Dileep, W. Dong, H. Mathys, D. A. Bennett, L. H. Tsai, M. Kellis, Epigenomic dissection of Alzheimer’s disease pinpoints causal variants and reveals epigenome erosion. Cell 186, 4422–4437.e21 (2023). DOI: https://doi.org/10.1016/j.cell.2023.08.040</p> <p>B. Frost, M. Hemberg, J. Lewis, M. B. Feany, Tau promotes neurodegeneration through global chromatin relaxation. Nat. Neurosci. 17, 357–366 (2014). DOI: https://doi.org/10.1038/nn.3639</p> <p>S. Baek, I. Goldstein, G. L. Hager, Bivariate genomic footprinting detects changes in transcription factor activity. Cell Rep. 19, 1710–1722 (2017). DOI: https://doi.org/10.1016/j.celrep.2017.05.003</p> <p>M. Bentsen, P. Goymann, H. Schultheis, K. Klee, A. Petrova, R. Wiegandt, A. Fust, J. Preussner, C. Kuenne, T. Braun, et al., ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Nat. Commun. 11, 4267 (2020). DOI: https://doi.org/10.1038/s41467-020-18035-1</p> <h3 id="running-the-code-for-single-cell-regulatory-occupancy-analysis">Running the code for Single-cell Regulatory Occupancy Analysis</h3> <div class="language-r highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="w">
</span><span class="n">Code</span><span class="w"> </span><span class="n">will</span><span class="w"> </span><span class="n">be</span><span class="w"> </span><span class="n">updated</span><span class="w"> </span><span class="n">here</span><span class="w"> </span><span class="n">soon</span><span class="w">

</span></code></pre></div></div> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">bioRxiv</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/PiD_2024.png" sizes="200px"> <img src="/assets/img/publication_preview/PiD_2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="PiD_2024.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="shi_single-nucleus_2024" class="col-sm-8"> <div class="title">Single-nucleus multi-omics identifies shared and distinct pathways in Pick’s and Alzheimer’s disease</div> <div class="author"> <em>Zechuan Shi</em>, Sudeshna Das, Samuel Morabito, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Emily Miyoshi, Jennifer Stocksdale, Nora Emerson, Shushrruth Sai Srinivasan, Arshi Shahin, Negin Rahimzadeh, Zhenkun Cao, Justine Silva, Andres Alonso Castaneda, Elizabeth Head, Leslie Thompson, Vivek Swarup' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2024.09.06.611761v1.full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Shi_PiD_08_12_24.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2024.09.06.611761"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.09.06.611761" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The study of neurodegenerative diseases, particularly tauopathies like Pick’s disease (PiD) and Alzheimer’s disease (AD), offers insights into the underlying regulatory mechanisms. By investigating epigenomic variations in these conditions, we identified critical regulatory changes driving disease progression, revealing potential therapeutic targets. Our comparative analyses uncovered disease-enriched non-coding regions and genome-wide transcription factor (TF) binding differences, linking them to target genes. Notably, we identified a distal human-gained enhancer (HGE) associated with E3 ubiquitin ligase (UBE3A), highlighting disease-specific regulatory alterations. Additionally, fine-mapping of AD risk genes uncovered loci enriched in microglial enhancers and accessible in other cell types. Shared and distinct TF binding patterns were observed in neurons and glial cells across PiD and AD. We validated our findings using CRISPR to excise a predicted enhancer region in UBE3A and developed an interactive database (http://swaruplab.bio.uci.edu/scROAD) to visualize predicted single-cell TF occupancy and regulatory networks.</p> </div> </div> </div> </li></ol> </div> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Zechuan Shi. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-publications",title:"publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-cv",title:"cv",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"nav-projects",title:"projects",description:"Ever tried. Ever failed. No matter. Try Again. Fail again. Fail better. -- Samuel Beckett",section:"Navigation",handler:()=>{window.location.href="/projects/"}},{id:"news-i-presented-my-research-talk-titled-single-nucleus-open-chromatin-accessibility-landscape-of-pick-s-and-alzheimer-s-disease-at-alzheimer-s-association-international-conference-2022-san-diego-ca",title:"I presented my research talk titled: \u201cSingle-nucleus open chromatin accessibility landscape of Pick\u2019s...",description:"",section:"News"},{id:"news-archrtosignac-an-object-conversion-package-that-i-developed-for-archr-to-signac",title:"ArchRtoSignac: an Object Conversion Package that I developed for ArchR to Signac.",description:"",section:"News"},{id:"news-successfully-advanced-in-phd-now-officially-a-phd-candidate-sparkles-smile-brain",title:'Successfully advanced in PhD; now officially a PhD Candidate! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> <img class="emoji" title=":brain:" alt=":brain:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f9e0.png" height="20" width="20">',description:"",section:"News"},{id:"news-i-presented-my-research-talk-titled-in-silico-module-perturbation-analysis-unlocks-a-functional-understanding-of-the-dynamic-gene-networks-in-single-cell-data-at-american-society-human-genetics-annual-meeting-2024-denvor-co",title:"I presented my research talk titled: \u201cIn Silico Module Perturbation Analysis unlocks a...",description:"",section:"News"},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"projects-bioinformatics-zero-to-one",title:"Bioinformatics - Zero to One",description:"These are my notes and practical guide to becoming a bioinformatic scientist",section:"Projects",handler:()=>{window.location.href="/projects/Bioinformatics_ZerotoOne/"}},{id:"projects-compact-module-perturbation-analysis",title:"compact - Module Perturbation Analysis",description:"Co-expression Module Perturbation Analysis in Cellular Transcriptomes",section:"Projects",handler:()=>{window.location.href="/projects/COMPACT_project/"}},{id:"projects-single-cell-regulatory-occupancy-archive-in-dementia",title:"Single-cell Regulatory Occupancy Archive in Dementia",description:"scROAD - A database offers comprehensive information on single-cell cCRE transcription factor occupancy data",section:"Projects",handler:()=>{window.location.href="/projects/scROAD_project/"}},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%7A%65%63%68%75%61%73@%75%63%69.%65%64%75","_blank")}},{id:"socials-orcid",title:"ORCID",section:"Socials",handler:()=>{window.open("https://orcid.org/0000-0002-2844-5816","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=QjZhFxIAAAAJ","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/rootze","_blank")}},{id:"socials-linkedin",title:"LinkedIn",section:"Socials",handler:()=>{window.open("https://www.linkedin.com/in/zechuan-tristan-shi-55b21387","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/ZeTristanShi","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>