<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> compact - Module Perturbation Analysis | Zechuan Shi </title> <meta name="author" content="Zechuan Shi"> <meta name="description" content="Co-expression Module Perturbation Analysis in Cellular Transcriptomes"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?29a31447c2ea9b76de9d010225a62982"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://rootze.github.io/projects/COMPACT_project/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Zechuan</span> Shi </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">compact - Module Perturbation Analysis</h1> <p class="post-description">Co-expression Module Perturbation Analysis in Cellular Transcriptomes</p> </header> <article> <h1 id="what-we-can-do-with-functional-annotated-clusters-in-single-cell-transcriptomes">What We Can Do with functional annotated clusters in single-cell transcriptomes</h1> <p>Biological functions are governed by gene regulatory networks that orchestrate a diverse array of dynamic cell states. These networks are altered throughout development, aging, disease, and in response to molecular stimuli, however such perturbations are exceedingly difficult to measure directly with technologies like single-cell RNA-seq (scRNA-seq). Here we propose a novel computational framework, CO-expression Module Perturbation Analysis for Cellular Transcriptomes (<strong>compact</strong>), to perform in-silico gene expression perturbations in single-cell data, and to track downstream changes in cell state dynamics. Building off of our previous method high-dimensional Weighted Gene Co-expression Network Analysis (hdWGCNA), <strong>compact</strong> applies direct perturbations to co-expression network hub genes, and uses the network structure to propagate the perturbation signal to other linked genes. This framework is highly flexible to perform knock-in, knock-down, or knock-out perturbations on different networks and sets of genes and in different cell lineages, allowing researchers to explore a wide range of strategies mimicking various experimental conditions and interventions. We demonstrate the efficacy of <strong>compact</strong> across established paradigms of cellular dynamic response, including mouse oligodendrocyte differentiation, human disease-associated microglia, and existing single-cell perturbation data. Notably, our in-silico perturbation simulations with <strong>compact</strong> have unveiled disease-relevant phenotypic outcomes resulting from co-expression module perturbations in human disease-associated microglia. These findings offer valuable insights into the molecular mechanisms underlying cellular states in disease, and identify novel targets for therapeutic intervention. In summary, our work introduces <strong>compact</strong> as a powerful tool for functionally dissecting gene network perturbations and elucidating their disease-relevant impact. By leveraging the wealth of information contained within single-cell transcriptomic data, <strong>compact</strong> facilitates comprehensive analyses of gene regulatory networks, paving the way for advancements in our understanding of cellular dynamics and disease pathology.</p> <p>Source: <code class="language-plaintext highlighter-rouge">vignettes/basic_tutorial.Rmd</code></p> <h1 id="introduction">Introduction</h1> <p>This tutorial covers the basics of using <strong><code class="language-plaintext highlighter-rouge">compact</code></strong> to perform module perturbation analysis on co-expression network/module on single-cell data.</p> <p>This tutorial covers the basics of using <strong><code class="language-plaintext highlighter-rouge">compact</code></strong> to perform in-silico pertubation experiments in single-cell transcriptomics data. For this tutorial we will use an integrated single-nucleus RNA-seq dataset of microglia from three different studies of Alzheimer’s disease, as we described previously in the <a href="https://www.sciencedirect.com/science/article/pii/S2667237523001273?via%3Dihub" rel="external nofollow noopener" target="_blank">hdWGCNA paper</a>.</p> <p>The in-silico perturbations that we demonstrate in this tutorial are based on gene co-expression modules, which are unbiased clusters of genes that are highly co-expressed in a gene-gene co-expression network. In order to run these pertubations, we first need to perform co-expression network analysis and group genes into modules by running <a href="https://smorabit.github.io/hdWGCNA/articles/basic_tutorial.html" rel="external nofollow noopener" target="_blank">hdWGCNA</a>. This has already been run on the tutorial microglia dataset, where we have identified four gene modules.</p> <h1 id="load-the-dataset">Load the dataset</h1> <p>Download the tutorial dataset from these Google Drive links:</p> <ul> <li><a href="https://drive.google.com/file/d/15-0URxGUgE2eCxx15zkoVwkkmLhZTFKH/view?usp=drive_link" rel="external nofollow noopener" target="_blank">Seurat object</a></li> <li><a href="https://drive.google.com/file/d/1BcnUx3eh4zHwk_JfBo8zpB70ewp0u1tX/view?usp=drive_link" rel="external nofollow noopener" target="_blank">co-expression adjacency matrix (.rda)</a></li> </ul> <p>First we load the required R libraries and the tutorial dataset. When loading the tutorial dataset, you need to update the filepath to the co-expression adjacency matrix (TOM) within the Seurat object (shown below).</p> <p>```{r eval=FALSE, echo=FALSE} library(tidyverse) library(cowplot) library(patchwork) library(Seurat) library(velocyto.R) library(hdWGCNA) library(compact) theme_set(theme_cowplot())</p> <h1 id="set-random-seed-for-reproducibility">set random seed for reproducibility</h1> <p>set.seed(12345)</p> <h1 id="load-the-processed-alzheimers-microglia-dataset">load the processed alzheimer’s microglia dataset</h1> <p>seurat_obj &lt;- readRDS(file=’AD_MG_hdWGCNA_COMPACT_tutorial.rds’)</p> <h1 id="update-the-path-to-the-co-expression-network-adjacency-matrix">update the path to the co-expression network adjacency matrix</h1> <p>net &lt;- GetNetworkData(seurat_obj) net$TOMFiles &lt;- ‘AD_MG_TOM.rda’ # it is better to use the absolute path seurat_obj &lt;- SetNetworkData(seurat_obj, net)</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
If you have not already done so, we need to construct a nearest-neighbors graph in the Seurat object. This is a key step in the typical single-cell clustering pipeline.

```{r eval=FALSE}
seurat_obj &lt;- FindNeighbors(
    seurat_obj,
    reduction='harmony',nearest neighbors?
    assay = 'RNA',
    annoy.metric = 'cosine'
)
</code></pre></div></div> <p>```{r eval=FALSE, echo=FALSE}</p> <h1 id="load-r-packages">load R packages</h1> <p>library(Seurat) library(ggplot2) library(cowplot) library(viridis) library(dplyr) library(tictoc) library(Matrix) library(hdWGCNA) library(COMPACT) theme_set(theme_cowplot()) library(viridis) library(ggrastr)</p> <h1 id="rna-velocity-packages-that-we-need-for-some-helper-functions">RNA velocity packages that we need for some helper functions</h1> <p>options(warn=-1) library(velocyto.R)</p> <h1 id="load-the-perturbation-functions">load the perturbation functions</h1> <p>source(‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Code/perturbation_functions.R’) source(‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Code/distance_plotting.R’) source(‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Code/compute_distance_test.R’)</p> <h1 id="load-spatial-plotting-functions-if-applicable">Load spatial plotting functions (if applicable)</h1> <p>source(‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Code/spatial_functions.R’)</p> <h1 id="setwd--setting-data-directory">setwd – setting data directory</h1> <p>setwd(‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Analysis/smorabit_hdWGCNA_2023/’)</p> <h1 id="output-directories">output directories</h1> <p>data_dir &lt;- ‘tutorial/data/’ fig_dir &lt;- ‘tutorial/figures/’</p> <p>dir.create(data_dir) dir.create(fig_dir)</p> <h1 id="re-load-mg-dataset">re-load MG dataset</h1> <p>seurat_obj &lt;- readRDS(file=’/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Analysis/smorabit_hdWGCNA_2023/data/microglia/AD_MG_hdWGCNA_COMPACT_tutorial.rds’)</p> <h1 id="update-the-path-to-the-hdwgcna-tom">update the path to the hdWGCNA TOM</h1> <p>net &lt;- GetNetworkData(seurat_obj) net$TOMFiles &lt;- ‘/dfs7/swaruplab/zechuas/Projects/Module_perturbation/Analysis/smorabit_hdWGCNA_2023/data/microglia/TOM/_ConsensusTOM-block.1.rda’ seurat_obj &lt;- SetNetworkData(seurat_obj, net)</p> <h1 id="make-a-cell-neighborhood-graph">make a cell neighborhood graph:</h1> <p>seurat_obj &lt;- FindNeighbors(seurat_obj, reduction=’harmony’)</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
Next we visualize the clusters in this dataset as a UMAP plot.

```{r eval=FALSE}

# Get the UMAP embedding from the seurat object
emb &lt;- Reductions(seurat_obj, 'umap')@cell.embeddings

# make a dataframe to plot with ggplot
plot_df &lt;- as.data.frame(emb)
plot_df$functional_anno &lt;- seurat_obj$functional_anno

# plot with ggplot
p &lt;- plot_df %&gt;%
  ggplot(aes(x=UMAP_1, y=UMAP_2, color=functional_anno)) +
  scale_color_viridis_d(option = "plasma") +  
  rasterise(geom_point(size=2, alpha=0.5), dpi = 300) +
  ggtitle("microglia functional_anno") +
  hdWGCNA::umap_theme()

p

</code></pre></div></div> <p><img src="figures/basic_tutorial/UMAP_microglia_functional_anno_MPA.png" width="1000" height="2000"></p> <h1 id="perform-in-silico-module-perturbation">Perform in-silico module perturbation</h1> <p>In this example, we will perform an in-silico knock-down for one of our microglia co-expression modules. First, let’s look at the expression level of these modules (module eigengene).</p> <p>```{r eval=FALSE}</p> <h1 id="add-module-expression-levels-mes-to-the-seurat-metadata">add module expression levels (MEs) to the seurat metadata</h1> <p>MEs &lt;- GetMEs(seurat_obj) meta &lt;- seurat_obj@meta.data seurat_obj@meta.data &lt;- cbind(meta, MEs)</p> <h1 id="dotplot">dotplot</h1> <p>p &lt;- DotPlot( seurat_obj, features=c(‘MG-M4’, ‘MG-M1’, ‘MG-M3’, ‘MG-M2’), group.by=’functional_anno’ ) + coord_flip() + scale_color_gradient2(high=’black’, mid=’grey75’, low=’grey99’) + theme( axis.text.x = element_blank(), axis.ticks.x = element_blank() ) + ylab(‘Trajectory →’) + xlab(‘’) + NoLegend()</p> <p>p</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
&lt;img src="figures/basic_tutorial/MG_module_dotplot.png" width="1000" height="1000"&gt;

The expression of microglia module M4 is concentrated at the end of the trajectory, within the activated microglia population. Next we will perform a knock-down experiment of this module, and we will predict how this perturbation will alter cellular states.

## Apply a knock-down using `ModulePerturbation`

Here we use the function `ModulePerturbation` to apply an in-silico knock-down of module MG-M4. This function consists of three main steps, which are all conveniently included in this function.

* Apply a primary perturbation to the hub genes of the co-expression module. For each hub gene, we perform zero-inflated count data regression to model the observed gene expression distribution. We simulate a perturbation by sampling from this distribution, then adding or subtracting the sampled counts to the observed counts for a knock-in or a knock-down respectively.

* Apply a secondary perturbation to all other genes in the co-expression module. The primary perturbation is propagated to other genes by exploiting the co-expression network structure. Genes with stronger connections in the network will exhibit a stronger perturbation.

* Calculate cell-cell transition probabilities based on the observed gene expression matrix and the perturbed gene expression matrix.

For this analysis we will perform the primary perturbation on the top 10 hub genes of module MG-M4. Let's check which genes we will be perturbing.

```{r eval=FALSE}

hub_genes &lt;- GetHubGenes(seurat_obj, n_hubs=10) %&gt;% subset(module == 'MG-M4')
hub_genes$gene_name
</code></pre></div></div> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>[1] "PTPN2"   "SLC11A1" "ACSL1"   "SPATA6"  "TBC1D14" "TMEM163" "ERC2"   
[8] "DPYD"    "CD163"   "TLR2"   
</code></pre></div></div> <p>```{r eval=FALSE}</p> <h1 id="provide-a-name-for-the-perturbation-which-will-be-stored-in-the-seurat-object">provide a name for the perturbation, which will be stored in the Seurat object</h1> <p>perturbation_name &lt;- ‘M4_down’</p> <h1 id="run-a-knock-down-of-the-top-10-hub-genes-from-mg-m4">run a knock-down of the top 10 hub genes from MG-M4</h1> <h1 id="should-take-2-3-min-to-run">should take 2-3 min to run</h1> <p>seurat_obj &lt;- ModulePerturbation( seurat_obj, mod = “MG-M4”, perturb_dir = -1, # negative indicates knock-down, positive indicates knock-in perturbation_name = perturbation_name, graph = ‘RNA_nn’, n_hubs = 10 )</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
The main ouputs of this function are a "perturbed" gene expression matrix, and a cell-cell transition matrix, which we can access from the Seurat object using the following code.

```{r eval=FALSE}
# get the perturbed expression matrix
X_per &lt;- GetAssayData(
  seurat_obj, assay=perturbation_name,
  layer='counts'
)

# get the cell-cell transition matrix
tp &lt;- Graphs(seurat_obj, paste0(perturbation_name, '_tp'))
</code></pre></div></div> <p>Next, let’s compare the observed and perturbed gene expression distributions for one of the MG-M4 hub genes, TMEM163. We expect that in the pertubed matrix, the expression of TMEM163 will be lower than in the observed matrix.</p> <p>```{r eval=FALSE}</p> <h1 id="generate-10-colors-from-the-plasma-palette">Generate 10 colors from the plasma palette</h1> <p>plasma_colors &lt;- viridis::viridis(10, option = “plasma”)</p> <h1 id="group-variable-for-vln-plots">group variable for vln plots</h1> <p>group.by &lt;- ‘functional_anno’</p> <h1 id="one-of-the-hub-genes-to-be-perturbed">one of the hub genes to be perturbed</h1> <p>cur_gene &lt;- ‘TMEM163’</p> <p>p1 &lt;- VlnPlot(seurat_obj, features = cur_gene, pt.size=0.1, group.by = group.by, assay=’RNA’) + scale_fill_manual(values = plasma_colors) + # Use the extracted colors here NoLegend() + xlab(‘’) + ylab(‘’) + ylim(0,5) + theme( axis.text.x = element_blank(), plot.margin=margin(c(0,0,0,0)) )</p> <p>p2 &lt;- VlnPlot(seurat_obj, features = cur_gene, pt.size=0.1, group.by = group.by, assay=perturbation_name) + scale_fill_manual(values = plasma_colors) + # Use the extracted colors here NoLegend() + ylab(‘’) + ylim(0,5) + theme( plot.margin=margin(c(0,0,0,0))) + ggtitle(‘’) + xlab(‘Functional Annotation / Pseudotime –&gt;’)</p> <p>p1 / p2</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
```{r eval=FALSE, echo=FALSE}
png(paste0(fig_dir, 'vln_compare_', perturbation_name, '_', cur_gene, '.png'), width=2000, height=1500, res=300)
print(p1 / p2)
dev.off()
</code></pre></div></div> <p><img src="figures/basic_tutorial/vln_compare_M4_down_TMEM163.png" width="2000" height="1500"></p> <h2 id="visualize-predicted-cell-transitions-after-perturbation">Visualize predicted cell transitions after perturbation</h2> <p>Here we will visualize the predicted cell-cell transitions after the in-silico knock-down of MG-M4. Similar to other methods like <a href="https://github.com/velocyto-team/velocyto.R" rel="external nofollow noopener" target="_blank">RNA Velocity</a> or <a href="https://github.com/morris-lab/CellOracle" rel="external nofollow noopener" target="_blank">CellOracle</a>, we visualize the inferred transitions as a vector field overlaid on a dimensionality reduction. We include the function <code class="language-plaintext highlighter-rouge">PlotTransitionVectors</code> to generate this plot. This function leverages some code from the R package <code class="language-plaintext highlighter-rouge">velocyto.R</code>, however in a future update we would like to remove this dependency since this package can be difficult to install on some machines.</p> <p>```{r eval=FALSE}</p> <p>p &lt;- PlotTransitionVectors( seurat_obj, perturbation_name = perturbation_name, reduction = ‘umap’, color.by=’pseudotime’, grid_resolution = 25 )</p> <p>p &lt;- p + NoLegend() + scale_color_gradientn(colors=viridis::plasma(256)) + coord_equal() + ggtitle(perturbation_name)</p> <p>p</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
&lt;img src="figures/basic_tutorial/vectorfield_M4_down_velocyto.png" width="2000" height="1500"&gt;

To generate the vector field, the cell-cell transition probabilities computed for this perturbation are used to infer the cells with similar expression profiles, and vectors are shown between these cells. In order to avoid overcrowding the plot, these vectors are averaged in a grid rather than showing a vector for each cell.

## Compare cell state similarity before and after perturbation

To further analyze the effect of the perturbation, we compute the E-distance between different functional annotations within the original and perturbed Seurat objects. The E-distance helps assess the similarity between different functional cell states present in the dataset.

First, we calculate energy distance between clusters in the observed gene expression matrix using the function `ComputeDistance`.

```{r eval=FALSE}

# Calculate similarity distance for original scRNA-seq data, method selected "edist" - energy distance
df_edist_original &lt;- ComputeDistance(
  seurat_obj,
  groupby = 'functional_anno',
  reduction = 'pca',
  method = "edist",
  verbose = TRUE  
)

# # Define your custom order
custom_order &lt;- levels(seurat_obj$functional_anno)

# Rearrange rows and columns of df_edist_observed based on the custom order
df_edist_original_reordered &lt;- df_edist_original[custom_order, custom_order]

# View the reordered dataframe
df_edist_original_reordered

</code></pre></div></div> <details> <summary> Output </summary> ``` # Homeostasis Surveillance1 Surveillance2 Surveillance3 # Homeostasis 0.000000 10.140967 17.220091 12.1832043 # Surveillance1 10.140967 0.000000 2.866413 3.1124290 # Surveillance2 17.220091 2.866413 0.000000 1.2850047 # Surveillance3 12.183204 3.112429 1.285005 0.0000000 # Surveillance4 8.723064 4.462216 3.529919 0.7661988 # Surveillance5 5.735601 7.660914 8.219343 3.6764471 # Activated1 13.835812 10.549585 7.522241 3.5081678 # Activated2 22.369534 18.084185 13.030198 8.7718675 # Activated3 17.792197 20.061153 17.297088 11.6027919 # Phagocytosis 26.392094 29.687218 25.688176 19.5139817 # Surveillance4 Surveillance5 Activated1 Activated2 Activated3 # Homeostasis 8.7230645 5.735601 13.835812 22.369534 17.792197 # Surveillance1 4.4622158 7.660914 10.549585 18.084185 20.061153 # Surveillance2 3.5299193 8.219343 7.522241 13.030198 17.297088 # Surveillance3 0.7661988 3.676447 3.508168 8.771868 11.602792 # Surveillance4 0.0000000 1.301993 1.908912 6.800320 8.081606 # Surveillance5 1.3019930 0.000000 2.087510 6.593177 5.495311 # Activated1 1.9089124 2.087510 0.000000 2.014579 3.216930 # Activated2 6.8003204 6.593177 2.014579 0.000000 1.950235 # Activated3 8.0816058 5.495311 3.216930 1.950235 0.000000 # Phagocytosis 15.4674423 12.337110 8.521408 5.171625 2.154714 # Phagocytosis # Homeostasis 26.392094 # Surveillance1 29.687218 # Surveillance2 25.688176 # Surveillance3 19.513982 # Surveillance4 15.467442 # Surveillance5 12.337110 # Activated1 8.521408 # Activated2 5.171625 # Activated3 2.154714 # Phagocytosis 0.000000 ``` </details> <p>Next, we perform a similar analysis on the perturbed Seurat assay ‘M4_down’</p> <p>```{r eval=FALSE}</p> <h1 id="set-the-default-assay-to-the-perturbed-assay-and-prepare-the-data">Set the default assay to the perturbed assay and prepare the data</h1> <p>seurat_obj_perturbed &lt;- seurat_obj # set up a new seurat object to avoid confusion</p> <p>DefaultAssay(seurat_obj_perturbed) &lt;- “M4_down”</p> <p>seurat_obj_perturbed &lt;- FindVariableFeatures(seurat_obj_perturbed, selection.method = “vst”, nfeatures = 2000) seurat_obj_perturbed &lt;- ScaleData(seurat_obj_perturbed)</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
Run PCA for the perturbed data and please remember to change the 'reduction.name'

```{r eval=FALSE}
seurat_obj_perturbed &lt;- RunPCA(seurat_obj_perturbed, features = VariableFeatures(object = seurat_obj_perturbed), reduction.name = "M4_down_pca")

seurat_obj_perturbed@reductions
</code></pre></div></div> <details> <summary> Output </summary> ``` &gt; seurat_obj_perturbed@reductions $umap A dimensional reduction object with key UMAP_ Number of dimensions: 2 Projected dimensional reduction calculated: FALSE Jackstraw run: FALSE Computed using assay: RNA $pca A dimensional reduction object with key PC_ Number of dimensions: 50 Projected dimensional reduction calculated: FALSE Jackstraw run: FALSE Computed using assay: RNA $harmony A dimensional reduction object with key harmony_ Number of dimensions: 50 Projected dimensional reduction calculated: TRUE Jackstraw run: FALSE Computed using assay: RNA $M4_down_pca A dimensional reduction object with key m4_down_pca_ Number of dimensions: 50 Projected dimensional reduction calculated: FALSE Jackstraw run: FALSE Computed using assay: M4_down ``` </details> <p>Compute similarity distance for the perturbed data</p> <p>```{r eval=FALSE}</p> <h1 id="computedistance">ComputeDistance</h1> <p>df_edist_perturbed &lt;- ComputeDistance(seurat_obj_perturbed, groupby = ‘functional_anno’, reduction = ‘M4_down_pca’, method = “edist”, verbose = TRUE)</p> <p>df_edist_perturbed</p> <h1 id="rearrange-rows-and-columns-of-df_edist_observed-based-on-the-custom-order">Rearrange rows and columns of df_edist_observed based on the custom order</h1> <p>df_edist_perturbed_reordered &lt;- df_edist_perturbed[custom_order, custom_order]</p> <h1 id="view-the-reordered-dataframe">View the reordered dataframe</h1> <p>df_edist_perturbed_reordered</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
&lt;details&gt; &lt;summary&gt; Output &lt;/summary&gt;
</code></pre></div></div> <blockquote> <p>df_edist_perturbed Phagocytosis Surveillance3 Surveillance2 Activated1 Surveillance1 Phagocytosis 0.0000000 4.1887166 5.3468811 1.9114647 6.367927 Surveillance3 4.1887166 0.0000000 0.3544971 0.8140641 0.702068 Surveillance2 5.3468811 0.3544971 0.0000000 1.6971191 0.444851 Activated1 1.9114647 0.8140641 1.6971191 0.0000000 2.354567 Surveillance1 6.3679270 0.7020680 0.4448510 2.3545673 0.000000 Surveillance4 3.3193247 0.2185912 0.8524824 0.3940233 1.108151 Activated2 1.6043523 1.2675249 2.0982363 0.3861708 2.864140 Surveillance5 2.9806149 0.5994561 1.5770397 0.4120399 1.629711 Homeostasis 5.0451679 1.9123500 2.8172922 2.0402917 1.892958 Activated3 0.9335111 1.9314490 3.0544205 0.5010339 3.670300 Surveillance4 Activated2 Surveillance5 Homeostasis Activated3 Phagocytosis 3.3193247 1.6043523 2.9806149 5.045168 0.9335111 Surveillance3 0.2185912 1.2675249 0.5994561 1.912350 1.9314490 Surveillance2 0.8524824 2.0982363 1.5770397 2.817292 3.0544205 Activated1 0.3940233 0.3861708 0.4120399 2.040292 0.5010339 Surveillance1 1.1081507 2.8641397 1.6297111 1.892958 3.6702999 Surveillance4 0.0000000 0.9458629 0.2065312 1.271465 1.2788325 Activated2 0.9458629 0.0000000 0.8679631 2.958652 0.3551624 Surveillance5 0.2065312 0.8679631 0.0000000 1.029319 0.9134280 Homeostasis 1.2714649 2.9586525 1.0293188 0.000000 2.7488030 Activated3 1.2788325 0.3551624 0.9134280 2.748803 0.0000000</p> </blockquote> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>&lt;/details&gt;

We visualize the similarity distance matrices for both the original and perturbed data using a heatmap. This will allow
us to compare the functional clusters' similarity or difference before and after the perturbation

```{r eval=FALSE}
#---------------------------
# NOTE plot heatmap

library(ggplot2)
library(reshape2)
library(scales)
library(patchwork)  # For arranging plots side by side


# # Example usage with default color palette
order_level &lt;- custom_order

# HeatmapDistance()
ht &lt;- HeatmapDistance(df_original = df_edist_original, df_perturbed = df_edist_perturbed, custom_order = order_level)

ht

</code></pre></div></div> <p>```{r eval=FALSE, echo=FALSE} png(paste0(fig_dir, ‘edistance_after_’, perturbation_name, ‘.png’), width=5000, height=2500, res=300) print(ht) dev.off()</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
&lt;img src="figures/basic_tutorial/edistance_after_M4_down.png" width="5000" height="2500"&gt;

We can see that the distance is smaller in the heatmap on the right, which indicates that the activated cells have become transcriptomically more similar to the homeostatic cells after our in-silco perturbation.

## Apply a knock-in

In this step, we will simulate another in-silico perturbation with `ModulePerturbation` to up-regulate the same co-expression module, and we will visualize the resulting transition vector field with `PlotTransitionVectors`. The only difference here is that we set `perturb_dir=1` for an up-regulation whereas previously we had `perturb_dir=-1` for a down-regulation.

```{r eval=FALSE}

# apply the up-regulation of module M4
perturbation_name &lt;- 'M4_up'
seurat_obj &lt;- ModulePerturbation(
    seurat_obj,
    mod = "MG-M4",
    perturb_dir = 1, # positive value indicates knock-in, all
    perturbation_name = perturbation_name,
    graph = 'RNA_nn',
    n_hubs = 10,
)

# plot the transition vectors
p1 &lt;- PlotTransitionVectors(
    seurat_obj,
    perturbation_name = "M4_up",
    reduction = 'umap',
    color.by='pseudotime',
    grid_resolution = 25,
    point_size=2 ) +
NoLegend() +
scale_color_gradientn(colors=viridis::plasma(256)) +
coord_equal() +
ggtitle("M4 knock-in")

# plot the transition vectors
p2 &lt;- PlotTransitionVectors(
    seurat_obj,
    perturbation_name = "M4_down",
    reduction = 'umap',
    color.by='pseudotime',
    grid_resolution = 25,
    point_size=2 ) +
NoLegend() +
scale_color_gradientn(colors=viridis::plasma(256)) +
coord_equal() +
ggtitle("M4 knock-down")

p1 | p2

</code></pre></div></div> <p><img src="figures/basic_tutorial/vectorfield_compare_MG-M4_velocyto.png" width="1000" height="1000"></p> <p>As expected, we can broadly see that the transition vectors indicate opposite affects of the knock-in and knock-down on the predicted cell states.</p> <h1 id="conclusion">Conclusion</h1> <p>Here we demonstrated the basics of the <strong><code class="language-plaintext highlighter-rouge">compact</code></strong> method to apply an in-silico knockdown of a the microglial activation module M4. By down-regulating this module, the method predicted that cells would shift to a more homeostatic-like state.</p> </article> <h2>References</h2> <div class="publications"> </div> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Zechuan Shi. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-publications",title:"publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-cv",title:"cv",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"nav-projects",title:"projects",description:"Ever tried. Ever failed. No matter. Try Again. Fail again. Fail better. -- Samuel Beckett",section:"Navigation",handler:()=>{window.location.href="/projects/"}},{id:"news-i-presented-my-research-talk-titled-single-nucleus-open-chromatin-accessibility-landscape-of-pick-s-and-alzheimer-s-disease-at-alzheimer-s-association-international-conference-2022-san-diego-ca",title:"I presented my research talk titled: \u201cSingle-nucleus open chromatin accessibility landscape of Pick\u2019s...",description:"",section:"News"},{id:"news-archrtosignac-an-object-conversion-package-that-i-developed-for-archr-to-signac",title:"ArchRtoSignac: an Object Conversion Package that I developed for ArchR to Signac.",description:"",section:"News"},{id:"news-successfully-advanced-in-phd-now-officially-a-phd-candidate-sparkles-smile-brain",title:'Successfully advanced in PhD; now officially a PhD Candidate! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> <img class="emoji" title=":brain:" alt=":brain:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f9e0.png" height="20" width="20">',description:"",section:"News"},{id:"news-i-will-present-my-research-talk-titled-in-silico-module-perturbation-analysis-unlocks-a-functional-understanding-of-the-dynamic-gene-networks-in-single-cell-data-at-american-society-human-genetics-annual-meeting-2024-denvor-co",title:"I will present my research talk titled: \u201cIn Silico Module Perturbation Analysis unlocks...",description:"",section:"News"},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"projects-bioinformatics-zero-to-one",title:"Bioinformatics - Zero to One",description:"These are my notes and practical guide to becoming a bioinformatic scientist",section:"Projects",handler:()=>{window.location.href="/projects/Bioinformatics_ZerotoOne/"}},{id:"projects-compact-module-perturbation-analysis",title:"compact - Module Perturbation Analysis",description:"Co-expression Module Perturbation Analysis in Cellular Transcriptomes",section:"Projects",handler:()=>{window.location.href="/projects/COMPACT_project/"}},{id:"projects-single-cell-regulatory-occupancy-archive-in-dementia",title:"Single-cell Regulatory Occupancy Archive in Dementia",description:"scROAD - A database offers comprehensive information on single-cell cCRE transcription factor occupancy data",section:"Projects",handler:()=>{window.location.href="/projects/scROAD_project/"}},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%7A%65%63%68%75%61%73@%75%63%69.%65%64%75","_blank")}},{id:"socials-orcid",title:"ORCID",section:"Socials",handler:()=>{window.open("https://orcid.org/0000-0002-2844-5816","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=QjZhFxIAAAAJ","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/rootze","_blank")}},{id:"socials-linkedin",title:"LinkedIn",section:"Socials",handler:()=>{window.open("https://www.linkedin.com/in/zechuan-tristan-shi-55b21387","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/ZeTristanShi","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>