<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Zechuan Shi </title> <meta name="author" content="Zechuan Shi"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?29a31447c2ea9b76de9d010225a62982"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://rootze.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Zechuan</span> Shi </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">bioRxiv</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/PiD_2024.png" sizes="200px"></source> <img src="/assets/img/publication_preview/PiD_2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="PiD_2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="shi_single-nucleus_2024" class="col-sm-8"> <div class="title">Single-nucleus multi-omics identifies shared and distinct pathways in Pick’s and Alzheimer’s disease</div> <div class="author"> <em>Zechuan Shi</em>, Sudeshna Das, Samuel Morabito, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Emily Miyoshi, Jennifer Stocksdale, Nora Emerson, Shushrruth Sai Srinivasan, Arshi Shahin, Negin Rahimzadeh, Zhenkun Cao, Justine Silva, Andres Alonso Castaneda, Elizabeth Head, Leslie Thompson, Vivek Swarup' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2024.09.06.611761v1.full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Shi_PiD_08_12_24.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2024.09.06.611761"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.09.06.611761" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The study of neurodegenerative diseases, particularly tauopathies like Pick’s disease (PiD) and Alzheimer’s disease (AD), offers insights into the underlying regulatory mechanisms. By investigating epigenomic variations in these conditions, we identified critical regulatory changes driving disease progression, revealing potential therapeutic targets. Our comparative analyses uncovered disease-enriched non-coding regions and genome-wide transcription factor (TF) binding differences, linking them to target genes. Notably, we identified a distal human-gained enhancer (HGE) associated with E3 ubiquitin ligase (UBE3A), highlighting disease-specific regulatory alterations. Additionally, fine-mapping of AD risk genes uncovered loci enriched in microglial enhancers and accessible in other cell types. Shared and distinct TF binding patterns were observed in neurons and glial cells across PiD and AD. We validated our findings using CRISPR to excise a predicted enhancer region in UBE3A and developed an interactive database (http://swaruplab.bio.uci.edu/scROAD) to visualize predicted single-cell TF occupancy and regulatory networks.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Immunity</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Tiwari_Immunity_2024.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Tiwari_Immunity_2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Tiwari_Immunity_2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tiwari_innate_2024" class="col-sm-8"> <div class="title">Innate immune training restores pro-reparative myeloid functions to promote remyelination in the aged central nervous system</div> <div class="author"> Vini Tiwari, Bharat Prajapati, Yaw Asare, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Alkmini Damkou, Hao Ji, Lu Liu, Nawraa Naser, Garyfallia Gouna, Katarzyna B. Leszczyńska, Jakub Mieczkowski, Martin Dichgans, Qing Wang, Riki Kawaguchi, Zechuan Shi, Vivek Swarup, Daniel H. Geschwind, Marco Prinz, Ozgun Gokce, Mikael Simons' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.cell.com/immunity/abstract/S1074-7613(24)00348-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Tiwari_Immunity_2024.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.immuni.2024.07.001"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.immuni.2024.07.001" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The reduced ability of the central nervous system to regenerate with increasing age limits functional recovery following demyelinating injury. Previous work has shown that myelin debris can overwhelm the metabolic capacity of microglia, thereby impeding tissue regeneration in aging, but the underlying mechanisms are unknown. In a model of demyelination, we found that a substantial number of genes that were not effectively activated in aged myeloid cells displayed epigenetic modifications associated with restricted chromatin accessibility. Ablation of two class I histone deacetylases in microglia was sufficient to restore the capacity of aged mice to remyelinate lesioned tissue. We used Bacillus Calmette-Guerin (BCG), a live-attenuated vaccine, to train the innate immune system and detected epigenetic reprogramming of brain-resident myeloid cells and functional restoration of myelin debris clearance and lesion recovery. Our results provide insight into aging-associated decline in myeloid function and how this decay can be prevented by innate immune reprogramming.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Nat Neurosci</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/oshea_2024.png" sizes="200px"></source> <img src="/assets/img/publication_preview/oshea_2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="oshea_2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="oshea_derivation_2024" class="col-sm-8"> <div class="title">Derivation and transcriptional reprogramming of border-forming wound repair astrocytes after spinal cord injury or stroke in mice</div> <div class="author"> Timothy M. O’Shea, Yan Ao, Shinong Wang, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Yilong Ren, Amy L. Cheng, Riki Kawaguchi, Zechuan Shi, Vivek Swarup, Michael V. Sofroniew' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Nature Neuroscience</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41593-024-01684-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/oshea_2024.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41593-024-01684-6"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41593-024-01684-6" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Central nervous system (CNS) lesions become surrounded by neuroprotective borders of newly proliferated reactive astrocytes; however, fundamental features of these cells are poorly understood. Here we show that following spinal cord injury or stroke, 90% and 10% of border-forming astrocytes derive, respectively, from proliferating local astrocytes and oligodendrocyte progenitor cells in adult mice of both sexes. Temporal transcriptome analysis, single-nucleus RNA sequencing and immunohistochemistry show that after focal CNS injury, local mature astrocytes dedifferentiate, proliferate and become transcriptionally reprogrammed to permanently altered new states, with persisting downregulation of molecules associated with astrocyte–neuron interactions and upregulation of molecules associated with wound healing, microbial defense and interactions with stromal and immune cells. These wound repair astrocytes share morphologic and transcriptional features with perimeningeal limitans astrocytes and are the predominant source of neuroprotective borders that re-establish CNS integrity around lesions by separating neural parenchyma from stromal and immune cells as occurs throughout the healthy CNS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Alzheimer’s &amp; Dementia</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/agudo_2024.png" sizes="200px"></source> <img src="/assets/img/publication_preview/agudo_2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="agudo_2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Agudo2024" class="col-sm-8"> <div class="title">BIN1K358R suppresses glial response to plaques in mouse model of Alzheimer’s disease</div> <div class="author"> Laura Fernandez Garcia‐Agudo, <em>Zechuan Shi</em>, Ian F. Smith, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Enikö A. Kramár, Katelynn Tran, Shimako Kawauchi, Shuling Wang, Sherilyn Collins, Amber Walker, Kai‐Xuan Shi, Jonathan Neumann, Heidi Yahan Liang, Celia Da Cunha, Giedre Milinkeviciute, Samuel Morabito, Emily Miyoshi, Narges Rezaie, Angela Gomez‐Arboledas, Adrian Mendoza Arvilla, Daryan Iman Ghaemi, Andrea J. Tenner, Frank M. LaFerla, Marcelo A. Wood, Ali Mortazavi, Vivek Swarup, Grant R. MacGregor, Kim N. Green' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Alzheimer’s &amp; Dementia</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://alz-journals.onlinelibrary.wiley.com/doi/full/10.1002/alz.13767" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/agudo2024.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1002/alz.13767"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/alz.13767" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The BIN1 coding variant rs138047593 (K358R) is linked to Late‐Onset Alzheimer’s Disease (LOAD) via targeted exome sequencing. To elucidate the functional consequences of this rare coding variant on brain amyloidosis and neuroinflammation, we generated BIN1K358R knock‐in mice using CRISPR/Cas9 technology. These mice were subsequently bred with 5xFAD transgenic mice, which serve as a model for Alzheimer’s pathology. The presence of the BIN1K358R variant leads to increased cerebral amyloid deposition, with a dampened response of astrocytes and oligodendrocytes, but not microglia, at both the cellular and transcriptional levels. This correlates with decreased neurofilament light chain in both plasma and brain tissue. Synaptic densities are significantly increased in both wild‐type and 5xFAD backgrounds homozygous for the BIN1K358R variant. The BIN1 K358R variant modulates amyloid pathology in 5xFAD mice, attenuates the astrocytic and oligodendrocytic responses to amyloid plaques, decreases damage markers, and elevates synaptic densities. BIN1 rs138047593 (K358R) coding variant is associated with increased risk of LOAD. BIN1 K358R variant increases amyloid plaque load in 12‐month‐old 5xFAD mice. BIN1 K358R variant dampens astrocytic and oligodendrocytic response to plaques. BIN1 K358R variant decreases neuronal damage in 5xFAD mice. BIN1 K358R upregulates synaptic densities and modulates synaptic transmission.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Molecular Neurodegeneration</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/tran_trem2r47h_2023.png" sizes="200px"></source> <img src="/assets/img/publication_preview/tran_trem2r47h_2023.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="tran_trem2r47h_2023.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran_trem2r47h_2023" class="col-sm-8"> <div class="title">A Trem2R47H mouse model without cryptic splicing drives age- and disease-dependent tissue damage and synaptic loss in response to plaques</div> <div class="author"> Kristine M. Tran, Shimako Kawauchi, Enikö A. Kramár, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Narges Rezaie, Heidi Yahan Liang, Jasmine S. Sakr, Angela Gomez-Arboledas, Miguel A. Arreola, Celia da Cunha, Jimmy Phan, Shuling Wang, Sherilyn Collins, Amber Walker, Kai-Xuan Shi, Jonathan Neumann, Ghassan Filimban, Zechuan Shi, Giedre Milinkeviciute, Dominic I. Javonillo, Katelynn Tran, Magdalena Gantuz, Stefania Forner, Vivek Swarup, Andrea J. Tenner, Frank M. LaFerla, Marcelo A. Wood, Ali Mortazavi, Grant R. MacGregor, Kim N. Green' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>Molecular Neurodegeneration</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://link.springer.com/article/10.1186/s13024-023-00598-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/tran_trem2r47h_2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s13024-023-00598-4"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13024-023-00598-4" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The TREM2 R47H variant is one of the strongest genetic risk factors for late-onset Alzheimer’s Disease (AD). Unfortunately, many current Trem2 R47H mouse models are associated with cryptic mRNA splicing of the mutant allele that produces a confounding reduction in protein product. To overcome this issue, we developed the Trem2R47H NSS (Normal Splice Site) mouse model in which the Trem2 allele is expressed at a similar level to the wild-type Trem2 allele without evidence of cryptic splicing products.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">bioRxiv</abbr> </div> <div id="Miyoshi2023" class="col-sm-8"> <div class="title">Spatial and single-nucleus transcriptomic analysis of genetic and sporadic forms of Alzheimer’s Disease</div> <div class="author"> Emily Miyoshi<sup>*</sup>, Samuel Morabito<sup>*</sup>, Caden M. Henningfield, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Negin Rahimzadeh, Sepideh Kiani Shabestari, Sudeshna Das, Neethu Michael, Fairlie Reese, Zechuan Shi, Zhenkun Cao, Vanessa Scarfone, Miguel A. Arreola, Jackie Lu, Sierra Wright, Justine Silva, Kelsey Leavy, Ira T. Lott, Eric Doran, William H. Yong, Saba Shahin, Mari Perez-Rosendahl, Elizabeth Head, Kim N. Green, Vivek Swarup' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2023.07.24.550282v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/miyoshi2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2023.07.24.550282"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.07.24.550282" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The majority of mammalian genes encode multiple transcript isoforms that result from differential promoter use, changes in exonic splicing, and alternative 3′ end choice. Detecting and quantifying transcript isoforms across tissues, cell types, and species has been extremely challenging because transcripts are much longer than the short reads normally used for RNA-seq. By contrast, long-read RNA-seq (LR-RNA-seq) gives the complete structure of most transcripts. We sequenced 264 LR-RNA-seq PacBio libraries totaling over 1 billion circular consensus reads (CCS) for 81 unique human and mouse samples. We detect at least one full-length transcript from 87.7% of annotated human protein coding genes and a total of \textbackslashtextasciitilde200,000 full-length transcripts, \textbackslashtextasciitilde40% of which have novel exon junction chains. To capture and compute on the three sources of transcript structure diversity, we introduce a gene and transcript annotation framework that uses triplets representing the transcript start site, exon junction chain, and transcript end site of each transcript. Using triplets in a simplex representation demonstrates how promoter selection, splice pattern, and 3′ processing are deployed across human tissues, with nearly half of multitranscript protein coding genes showing a clear bias toward one of the three diversity mechanisms. Evaluated across samples, the predominantly expressed transcript changes for 74% of protein coding genes. In evolution, the human and mouse transcriptomes are globally similar in types of transcript structure diversity, yet among individual orthologous gene pairs, more than half (57.8%) show substantial differences in mechanism of diversification in matching tissues. This initial large-scale survey of human and mouse long-read transcriptomes provides a foundation for further analyses of alternative transcript usage, and is complemented by short-read and microRNA data on the same samples and by epigenome data elsewhere in the ENCODE4 collection.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">STAR Protoc</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/starprotoc_2022.gif" sizes="200px"></source> <img src="/assets/img/publication_preview/starprotoc_2022.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="starprotoc_2022.gif" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Shi_2022" class="col-sm-8"> <div class="title">Protocol for single-nucleus ATAC sequencing and bioinformatic analysis in frozen human brain tissue</div> <div class="author"> <em>Zechuan Shi<sup>*</sup></em>, Sudeshna Das<sup>*</sup>, Samuel Morabito, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Emily Miyoshi, Vivek Swarup' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>STAR Protocols</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/pii/S2666166722003719" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/shi_star_protocol_2022.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.xpro.2022.101491"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xpro.2022.101491" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Single-nucleus ATAC sequencing (snATAC-seq) employs a hyperactive Tn5 transposase to gain precise information about the cis-regulatory elements in specific cell types. However, the standard protocol of snATAC-seq is not optimized for all tissues, including the brain. Here, we present a modified protocol for single-nuclei isolation from postmortem frozen human brain tissue, followed by snATAC-seq library preparation and sequencing. We also describe an integrated bioinformatics analysis pipeline using an R package (ArchRtoSignac) to robustly analyze snATAC-seq data. For complete details on the use and execution of this protocol, please refer to Morabito et al. (2021).</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Pedosphere</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ma_elevated_2018.gif" sizes="200px"></source> <img src="/assets/img/publication_preview/ma_elevated_2018.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ma_elevated_2018.gif" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="ma_elevated_2018" class="col-sm-8"> <div class="title">Elevated CO2 Accelerates Phosphorus Depletion by Common Bean (Phaseolus vulgaris) in Association with Altered Leaf Biochemical Properties</div> <div class="author"> Zhong Ma, Jennifer Flynn, Grant Libra, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Zechuan Shi' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Pedosphere</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/pii/S100201601760420X" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/S1002-0160(17)60420-X"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/S1002-0160(17)60420-X" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Phosphorus (P) is a major limiting factor for plant productivity in many ecosystems and agriculture. The projected increase in atmospheric CO2 is likely to result in changes in plant mineral consumption and growth. We studied P depletion by common bean (Phaseolus vulgaris) cultured hydroponically under ambient (377 ± 77 μmol mol−1) or elevated (650 ± 32 μmol mol−1) CO2 in media of low or high P. Under elevated CO2 compared to ambient CO2, the maximum P depletion rate increased by 98% at low P and 250% at high P, and P was depleted about 2–5 weeks sooner; leaf acid phosphatase (APase) activity and chlorophyll content both increased significantly; root-to-shoot ratio increased significantly at high P, although it was unaffected at low P; lateral root respiration rate showed no change, suggesting that CO2 did not affect P depletion via metabolic changes to the roots; the total biomass at final harvest was significantly higher at both low and high P. Our data showed that the increased rate and amount of P depletion during plant growth under elevated CO2 occurred in association with alterations in leaf biochemical properties, i.e., enhanced activities of leaf APase and increased leaf chlorophyll content.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Zechuan Shi. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-publications",title:"publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-cv",title:"cv",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"nav-projects",title:"projects",description:"Ever tried. Ever failed. No matter. Try Again. Fail again. Fail better. -- Samuel Beckett",section:"Navigation",handler:()=>{window.location.href="/projects/"}},{id:"news-i-presented-my-research-talk-titled-single-nucleus-open-chromatin-accessibility-landscape-of-pick-s-and-alzheimer-s-disease-at-alzheimer-s-association-international-conference-2022-san-diego-ca",title:"I presented my research talk titled: \u201cSingle-nucleus open chromatin accessibility landscape of Pick\u2019s...",description:"",section:"News"},{id:"news-archrtosignac-an-object-conversion-package-that-i-developed-for-archr-to-signac",title:"ArchRtoSignac: an Object Conversion Package that I developed for ArchR to Signac.",description:"",section:"News"},{id:"news-successfully-advanced-in-phd-now-officially-a-phd-candidate-sparkles-smile-brain",title:'Successfully advanced in PhD; now officially a PhD Candidate! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> <img class="emoji" title=":brain:" alt=":brain:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f9e0.png" height="20" width="20">',description:"",section:"News"},{id:"news-i-will-present-my-research-talk-titled-in-silico-module-perturbation-analysis-unlocks-a-functional-understanding-of-the-dynamic-gene-networks-in-single-cell-data-at-american-society-human-genetics-annual-meeting-2024-denvor-co",title:"I will present my research talk titled: \u201cIn Silico Module Perturbation Analysis unlocks...",description:"",section:"News"},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"projects-compact-module-perturbation-analysis",title:"COMPACT - Module Perturbation Analysis",description:"Co-expression Module Perturbation Analysis in Cellular Transcriptomes",section:"Projects",handler:()=>{window.location.href="/projects/COMPACT_project/"}},{id:"projects-single-cell-regulatory-occupancy-archive-in-dementia",title:"Single-cell Regulatory Occupancy Archive in Dementia",description:"scROAD - A database offers comprehensive information on single-cell cCRE transcription factor occupancy data",section:"Projects",handler:()=>{window.location.href="/projects/scROAD_project/"}},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%7A%65%63%68%75%61%73@%75%63%69.%65%64%75","_blank")}},{id:"socials-orcid",title:"ORCID",section:"Socials",handler:()=>{window.open("https://orcid.org/0000-0002-2844-5816","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=QjZhFxIAAAAJ","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/rootze","_blank")}},{id:"socials-linkedin",title:"LinkedIn",section:"Socials",handler:()=>{window.open("https://www.linkedin.com/in/zechuan-tristan-shi-55b21387","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/ZeTristanShi","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>